src/hosted-fields/internal/index.js

```diff
@@ -92,6 +92,7 @@ function makeMockInput(name) {
   input.type = 'text';
   input.name = name;
   input.setAttribute('autocomplete', constants.autocompleteMappings[name]);
+  input.setAttribute('inputmode', constants.inputmodeMappings[name]);
```

Comment:
This input is hidden to accommodate autofill and should never be accessed from the user, so no need to add the attribute here.